ts/client/features/transfer.ts

```diff
@@ -480,6 +516,7 @@ export class SimulatorPool extends OffchainTransferFactory
 
     getNextPipe() {
         const source = this.genTx();
+        // TODO Remove this check?
         if (this.tokenID === undefined) throw new Error("tokenID not set");
```

Comment:
If `tokenID` is always set in constructor (https://github.com/thehubbleproject/hubble-contracts/pull/556/files#diff-f8e595b0b3744ed7278b7e180945494868c328b812257e8f93251f538bef57d3R485) we should be able to remove this safety check